command/filemanager.go

```diff
@@ -53,3 +60,133 @@ func (fm *FileManager) CopyInlineFile(name string, fileContent pulumi.StringInpu
 			Sudo:   useSudo,
 		}, opts...)
 }
+
+// CopyRelativeFolder copies recursively a relative folder to a remote folder.
+// The path of the folder is relative to the caller of this function.
+// For example, if this function is called from ~/foo/test.go with remoteFolder="bar"
+// then the full path of the folder will be ~/foo/bar“.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is the result if we call from `~/foo/test.go` with relative folder `bar/toto` and remoteFolder `tutu` ?
Isn't copy always happening in `tmp` remote folder ?

Reply:
It means you are trying to copy files from `~/foo/bar/toto` to `tutu` which means files in `~/foo/bar/toto` will be copied into `tutu/toto` (I did my best to make this function behaves the same as `cp -r`).